frontend/src/Components/MyHistory/SingleDayTasksContainer.tsx

```diff
@@ -7,27 +7,29 @@ const SingleDayTasksContainer = (props: {
   date: Date;
   historyTasks: TaskInterface[];
 }) => {
-  const filteredTasks = props.historyTasks.filter(
-    (task) => task.deliveryTime.toDateString() === props.date.toDateString()
-  );
+  console.log(Array.isArray(props.historyTasks));
+  console.log("props.historyTasks:", props.historyTasks);
+  console.log(props.date);
 
+  if (props.historyTasks.length > 0){
   return (
     <Box sx={{display: "flex", flexDirection: "column"}}>
       <HistoryTaskDate date={props.date} />
-      {filteredTasks.map((task) => (
+      {props.historyTasks.map((task) => (
         <Box key={task.id} sx={{ display: "flex", mb: 1, p: 1, bgcolor: "#FFFFFF", border: "1px solid #DFDFDF", borderRadius: 2, flexDirection: "column", justifyContent: "flex-start" }}>
-          <Typography sx={{ mr: 1 }}>{task.name}</Typography>
+           <Typography sx={{ mr: 1 }}>{task.id}</Typography>  {/*USING ID FOR NOW */}
```

Comment:
why is it using the ID for now?

Reply:
updated to use volunteer name if volunteer exists. otherwise we use id